docs/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+[EVG-<number>](https://jira.mongodb.org/browse/EVG-<number>)
```

Comment:
I just added an [autolink reference](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls) to this repo so JIRA links will be caught automatically. I propose replacing this with `EVG-NNNNN` so that we can just double click on `NNNNN` to highlight it (rather than having to click and drag to select the brackets)

```suggestion
EVG-NNNNN
```

Reply:
this is super cool :O